react-app/src/Components/Title.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Title.css';
+import Uwlogo from '../imagelogouw.png'
+
+function Title() {
+  return (
+    <div className="Title">
```

Comment:
This is great. You can reduce the number of elements within this component if you'd like.

```
<h1 class='Title'>
   University of Waterloo Concept App
  <img src={Uwlogo} width="100" height="100" alt="uwlogo" />
</h1>
```

Should be sufficient for what you want.